### ISSUE TITLE
Stuck when using or screen-off

### ISSUE THREAD

**Issue by chhua** [LinusHenze/Fugu14#141]
Need to double click screen many times or keep trying press power button wake screen on, anyone have same occur? Or just my machine issue.

Dev: iPhone 12
iOS: 14.4
u0: v7.0.1

**Comment by racerx8967** [comment]
The new version does not fix this. Yes I fs restored, yes I’ve updated then restored again etc etc. I’ve tried all steps. It still persists. iPhone 11 Pro Max 14.3

**Comment by racerx8967** [comment]
I’d also like to know that it does not do this when it’s plugged into the computer or charging.

**Comment by jaygee562** [comment]
> I’d also like to note that it does not do this when it’s plugged into the computer or charging. 

you are for sure using the newest jailbreak through altsore?

**Comment by jaygee562** [comment]
the exploit process is done through altstore not fugu app

**Comment by racerx8967** [comment]
Welp here what I did. I installed Fugu14 last night, I then saw there was an update today. First I tried restorefs just updating through alt store. That didn’t work. Then I thought ok I’ll do a restorefs then download the new version from GitHub compile it put it on and then install alt store then I’ll be set. That didn’t work. I’m not quite sure what much else to do. I will wipe my Mac downloads folder restore one more time in order to be completely sure. I will update here once finished.

**Comment by jaygee562** [comment]
you don't need a mac or compiler for the new version of unc0ver and altstore
the only thing you need is altstore on your phoneeee

**Comment by jaygee562** [comment]
you don't need to sideload or compile fugu14, all that is fone through altstore, you messed up by restoring your rootfs

**Comment by racerx8967** [comment]
That makes no sense. Restorefs is just setting the phone back to the way it was like the untether was never installed.. 

What you’re purposing is just installing alt store then installing uncover?

**Comment by jaygee562** [comment]
yes. please report back

**Comment by racerx8967** [comment]
Ok I understand what I did wrong. The problem is my alt store doesn’t pop up and say “do you want to untether?” It just side loads unc0ver. That’s why I wasn’t understanding what was wrong. I wonder why it’s not asking me to untether

**Comment by jaygee562** [comment]
did you solve?

**Comment by racerx8967** [comment]
Nope. Still looking into why it’s not prompting me to install the untether on AltStore. Extremely nerve wracking. I see a reddit thread saying the same thing and have begun to look through filza for tenant traces of the first version of untether and remove them. But have yet to find any. 

Any ideas on why it’s not asking me to install the untether?

**Comment by jaygee562** [comment]
did you update alt server on your pc?

**Comment by jaygee562** [comment]
> Nope. Still looking into why it’s not prompting me to install the untether on AltStore. Extremely nerve wracking. I see a reddit thread saying the same thing and have begun to look through filza for tenant traces of the first version of untether and remove them. But have yet to find any.
> 
> Any ideas on why it’s not asking me to install the untether?

upon looking at the main.swift source code, look here
![set_true](https://user-images.githubusercontent.com/53057371/139518135-c4eb33dd-3c9e-4470-bb6a-8a233f41498d.jpg)

**Comment by jaygee562** [comment]
perhaps compiling fugu 14 w this value set to "true" will solve it? but please refer to issue #142 
me and some other guy came up w a solution for another problem via editing the source code under main.swift

**Comment by racerx8967** [comment]
That’s exactly what I was looking for in filza. Thank you. I don’t know how you found that so fast. Yes alt server was updated.  Let me give this a true I’m certain there’s a check somewhere set to true saying that fugus there already even though it’s not. I have commented on 142 and your fix I had to implement to get the first version to work. I appreciate all your help. Lemme try this and I’ll let ya know how it works out.

**Comment by racerx8967** [comment]
No dice. Still no prompt in AltStore when trying the above. Just to be sure. The steps would be:

install alt store
Go to uncover page and click open with alt store
Sign in alt store 
Alt store signs uncover and SHOULD PROMPT to install untether...

that’s it correct? That’s all I should have to do?

I wonder if there’s a command I can run in terminal to complete remove fugu14 

also I appreciate you taking the time to help me figure this puzzle out.

**Comment by jaygee562** [comment]
so let me get this straight. you are already jailbroken or not? :)

**Comment by jaygee562** [comment]
yes all you have to do with the new altstore is side-load unc0ver from unc0ver.dev, after clicking "open in altstore" the prompts should be activated like this tweet https://twitter.com/altstoreio/status/1453882557628502019?s=21

**Comment by jaygee562** [comment]
i take it you were previously jailbroken on old fugu exploit and old unc0ver, correct?

**Comment by racerx8967** [comment]
I was already jailbroken tethered with unc0ver. 

I installed Fugu14 before the update and then started having the screen wake issue. 

Today I saw the update and compiled the néw fugu. Once I saw that wasn’t the way to do it I tried to set everything back via fsrestore and then install alt store again and side load unc0ver. Now it doesn’t prompt to install the untether.  

Wether it’s because I had fugu installed already or it’s a bug with iOS 14.3 and 11 pro Max I’m still unsure. I’ve tried all of our ideas and still no prompts. I was just gonna future restore but I don’t believe the current sep is compatible with 14.3 so that’s outta the question as well.

**Comment by chhua** [comment]
I use the latest version of u0 altserver, still stuck and auto reboot oftenly.. Today I'm trying restore my iOS setting and wipe everything then jailbreak again, keep updating.

**Comment by racerx8967** [comment]
![963756F2-9739-4EEA-BD1F-714315E62B29](https://user-images.githubusercontent.com/21145028/139520544-d9922bec-0aec-46e4-9392-69f623659477.png)


I’m looking for the plist that contains the placeholder for the install. Should be able to tell AltStore to trigger the pop up question is just where is it.

**Comment by jaygee562** [comment]
have you tried opening it with a text editor?

**Comment by racerx8967** [comment]
Yeah still poking around. Found an interesting Lil bit in the code  http://unc0ver.dev/fugu14app 

I assume this is a payload file that is supposed to trigger alt store. Doesn’t do anything for me except install and not run and still won’t trigger AltStore. Still workin on it. Any ideas please throw them my way.
